src/main/java/com/tikal/jenkins/plugins/multijob/SubTask.java

```diff
@@ -8,7 +8,6 @@
 import hudson.model.AbstractProject;
 import hudson.model.Cause.UpstreamCause;
 
-import java.util.ArrayList;
```

Comment:
It's better to avoid unrequired changes to avoid merge conflicts with other PRs.